tests/project/build.tests.ts

```diff
@@ -1,27 +1,52 @@
-/**
- * NOTE: This test deletes ./build and ./darkreader.js
- *       (but does not run automatically)
- */
-import {forka} from './utils';
-import {resolve, join} from 'path';
+import {timeout} from '../support/test-utils';
+import {childClosed, watchStream} from './utils';
+import {join, resolve} from 'path';
 import {tmpdir} from 'os';
-import {mkdtemp, rm} from 'fs/promises';
+import {mkdtemp} from 'fs/promises';
+import {fork} from 'child_process';
 
 const rootPath = (path: string) => resolve(__dirname, '../..', path);
 const buildModule = rootPath('tasks/build.js');
 
-describe.skip('build.js', () => {
-    it('Builds from a different dir', async () => {
-        // Build from temp dir instead of root dir
-        const cwd = await mkdtemp(join(tmpdir()));
-        await rm(rootPath('build'), {recursive: true, force: true});
-        await rm(rootPath('darkreader.js'), {force: true});
+describe('tasks/build-js.js', () => {
+    it('should respond to SIGINT signals by exiting immediately', async () => {
+        expect.assertions(3);
+        const tmpDir: string = await mkdtemp(join(tmpdir()));
+        const wantToWaitNoMoreThan = 500;
+
+        // Execute build
+        // Give it a chance to reach compilation stage
+        const child = fork(buildModule, ['--debug'], {cwd: tmpDir, silent: true});
+
+        // Wait for "clean" step to complete
+        await watchStream(child.stdout).forMatch(/ clean \(\d+ms\)/);
+        // Let "bundle-js" to get started
+        await timeout(500);
+
+        // Fire!
+        child.kill('SIGINT');
+
+        // Wait for close
+        await childClosed(child, {timeout: wantToWaitNoMoreThan});
+
+        expect(child.killed).toBeTrue();
+        expect(child.exitCode).toBeGreaterThan(0);
+
+        // By now, "bundle-js" step should not have completed normally
+        expect((child.stdout.read() || '')).not.toMatch(/ bundle-js \(\d+ms\)/);
+    });
+});
+
+describe('tasks/build.js', () => {
+    // Slow test (skipped by default)
+    it.skip('should build successfully from an unexpected working directory', async () => {
+        const tmpDir: string = await mkdtemp(join(tmpdir()));
 
         // Fire
-        const {code, stdout} = await forka(buildModule, ['--debug', '--release', '--api'], {cwd});
+        const child = fork(buildModule, ['--debug', '--release', '--api'], {cwd: tmpDir, silent: true});
```

Comment:
Wait... a debug release api?!?

Reply:
Originally I thought it would conflict, especially the initial clean step perhaps targeting the same directory. But they only attend to the directories they write. Very nice :-)